bigquery/datasets.rb

```diff
@@ -14,10 +14,9 @@
 
 def create_bigquery_client project_id:
   # [START create_bigquery_client]
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside the create_bigquery_client function which has a project_id parameter.